### DISCUSSION TITLE
A way to stop updates using crosh?

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#1053] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
While I am in skiovox i can use crosh with the chrome-untrusted://crosh and i was wanting code that stops updates perm.

**Comment by S-PScripts** [comment]
can't you use caub https://caub.vercel.app/ or caub flags https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Chrome%20Auto%20Update%20Blocker%202

**Reply by CGRmags** [reply]
no both dont work
